freq.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2016 Eric Barkie. All rights reserved.  // Use of this source code is governed by the MIT license
+// that can be found in the LICENSE file.
+
+package aprs
+
+import (
+	"fmt"
+)
+
+// Freq contains an APRS FreqSpec frequency report
+// http://www.aprs.org/info/freqspec.txt
+type Freq struct {
+	Mhz    float64 // Frequency in Mhz
+	Tone   int     // Tone in hz
+	CTCSS  int     // ctcss (mutually exclusive with tone and dcs)
+	DCS    int     // dcs (mutually exclusive with tone and ctcss)
+	Offset int     // +/- offset in mhz
+	Range  int     // range in miles
+	Narrow bool    // defaults to false for wideband
+}
+
+// Render renders the frequency struct into a byte-slice
+func (f *Freq) Render() []byte {
```

Comment:
yeah.. reading the spec I was fooled into thinking []byte was going to be the way to go, so made a pattern of returning []byte from everything, but with the benefit of hindsight, you never really need to do anything other than concatenate strings. What do you think about refactoring ALL of this (freq.go AND position.go) to return string natively?

Reply:
Yup, I totally understand...I remember when I was first working on this package I bounced between the two types a lot to try to balance consistency with ease of use (read: being able to use `fmt`). 

I think `freq.go` looks so much nicer with the changes you made and yes, taking a closer look `position.go` it looks like that could benefit from the same treatment if you're up for that.

Reply:
Okidokee I rewired the rest of it to use string, and got rid of the exported Byte() method. Go test is throwing some warnings at me about multiple `main()` definitions from the examples directory so I threw up a PR to nest those; if that works for you I'll probably follow this up with an example for position send before we merge all this in. I'm also going to resolve the ensuing comment and keep this one open for dialog.

Reply:
Awesome stuff.  I have no concerns with any of this...

I'm going to go ahead and merge the other PR since it's independent to this.  When you've got the position example(s) in a state you're happy with let me know and we'll merge this as well.

Reply:
Ok I think we're ready.  I merged down the nested examples, added one of my own for position reports, and fixed the symbol attrib to be a simple string rather than []string{} which still felt like it was bending-over-backwards.  Thanks for making this library btw it's very well done :relaxed:

Reply:
I'm very happy you've found it useful and thanks for all the work on the position report!